src/MyEventEmitter.js

```diff
@@ -1,12 +1,87 @@
 'use strict';
 
-export class MyEventEmitter {
-  on() {}
-  once() {}
-  off() {}
-  emit() {}
-  prependListener() {}
-  prependOnceListener() {}
-  removeAllListeners() {}
-  listenerCount() {}
+class MyEventEmitter {
+  constructor() {
+    this.events = {};
+  }
+
+  on(eventName, callBack) {
+    if (!this.events[eventName]) {
+      this.events[eventName] = [];
+    }
+
+    this.events[eventName].push(callBack);
+  }
+
+  once(eventName, callBack) {
+    const onceWrapper = (...args) => {
+      callBack.apply(this, args);
+
+      this.off(eventName, onceWrapper);
```

Comment:
Removing the listener immediately after being called might lead to an issue, as it might not be removed if there are multiple similar listeners.

Reply:
In my opinion it still should work fine for all cases. (Probably, I just didn't fully get your comment). To prove that my solution works well, I left an example below the main code.